app/api/user/photo/route.ts

```diff
@@ -0,0 +1,19 @@
+import { put } from '@vercel/blob'
+import { NextResponse } from 'next/server'
+
+export async function POST(request: Request): Promise<NextResponse> {
+  const { searchParams } = new URL(request.url)
+  const filename = searchParams.get('filename')
+
+  if (!filename || !request.body) {
+    return new NextResponse('Error: Invalid filename or request body.', {
```

Comment:
**nitpick:** If it's a generic file upload route, we should keep it in `app/api/files/route.ts`